core/sql/parser/sqlparser.y

```diff
@@ -16026,13 +16027,30 @@ exe_util_init_hbase : TOK_INITIALIZE TOK_TRAFODION
 
 		 DDLExpr * de = new(PARSERHEAP()) DDLExpr(TRUE, FALSE, TRUE, FALSE,
                                                           FALSE, FALSE,
-							  FALSE, FALSE, FALSE,
+							  FALSE, FALSE, FALSE, FALSE,
```

Comment:
I agree. Cleaning this up is a good idea. I will do it under a separate JIRA. I have written JIRA TRAFODION-2111 for this purpose.